src/colony/modules/net.js

```diff
@@ -163,7 +147,17 @@ TCPSocket.prototype.connect = function (/*options | [port], [host], [cb]*/) {
 
   function setUpConnection(ip) {
     if (self.socket == null) {
-      if (self._secure) ensureSSLCtx();
+      if (self._secure) {
+        var custom_certs = null;
+        self._ssl_checkCerts = (opts.rejectUnauthorized !== false);
+        if (opts.ca) custom_certs = opts.ca.map(function (pem_data) {
+            // TODO: review PEM specs and axTLS needs; make more thorough if needed
+            return Buffer(pem_data.toString().split('\n').filter(function (line) {
```

Comment:
My biggest concern would be how this handles invalid Base64 data, i.e. is a thrown exception okay (probably, but maybe we can call it out)